pages/md-style-guide.md

```diff
@@ -1,191 +1,279 @@
 ---
-title: Markdown Style Guide
+title: JSON Schema documentation style guide
```

Comment:
Let's rename the file by docs-style-guide.md and let's move it here:

````
/pages/contributing/
    ├── docs-style-guide.md         # Docs contributing guidelines
````

If everyone agrees I can make the change as last step. To move it there we need to add a typescript page as well render the md as styled webpage.